### PR TITLE
Locally running Rasa X

### PR DESCRIPTION
**Pull request by ricwo**

### CODE REVIEW

**Comment by tmbo** [comment]
@ricwo is this ready for review?

**Comment by ricwo** [comment]
@tmbo not quite yet. there are two todos you marked which I haven't addressed yet (endpoint config and auth token)

File: rasa/cli/up.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+def start_event_service():
+    from rasa_platform.services.event_service import main
+    main("rasa_event.log")
+
+
+def up(args: argparse.Namespace):
+    from rasa_core.utils import print_success
+    print_success("Starting Rasa Core")
+    from rasa_platform import config
```
Comment (tmbo) [inline rasa/cli/up.py]: this needs to be wrapped in try/catch: the platform might not be installed and if it is not this should fail with a proper error message

**Comment by ricwo** [comment]
@tmbo ready for review

**Pull request merged**